gocourse02/main.go

```diff
@@ -0,0 +1,25 @@
+package main
+
+import (
+	"fmt"
+	zoo "gocourse02/zoo"
+)
+
+func main() {
+	lion := zoo.NewLion(zoo.NewAnimal("Mykola", zoo.Male), "green")
+	lioness := zoo.NewLion(zoo.NewAnimal("Natalia", zoo.Female), "white")
+	cub := lion.Reproduce(*lioness)
+	cub.Show()
+
+	zooKeeper := zoo.NewZookeeper("Mykyta", []zoo.Reproducer{})
+	zooKeeper.CatchAnimal(lion.Animal)
+	zooKeeper.CatchAnimal(lioness.Animal)
+	zooKeeper.CatchAnimal(cub.Animal)
```

Comment:
Чи можемо трішки переробити параметри методу, щоб ось так працювало?
```suggestion
	zooKeeper.CatchAnimal(lion)
	zooKeeper.CatchAnimal(lioness)
	zooKeeper.CatchAnimal(cub)
```